README.md

```diff
@@ -34,6 +34,7 @@ Refer to the Cortex Certifai documentation: https://cognitivescale.github.io/cor
 | [interpreting_certifai_fairness_robustness_scores.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/interpreting_scores/notebooks/interpreting_fairness_robustness_scores) - Examination of robustness and fairness on simulated datasets | This notebook uses binary classification models trained on simple simulated datasets to build intuition around the robustness and fairness scores. | Binary classification | Multilayer Perceptron (MLP) | Fairness <br /> Performance-Accuracy  <br /> Robustness | interpreting scores |
 | [MulticlassHeartDisease.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/multiclass) | Illustrates the use of Certifai to create and run a scan on local multiclass classification models that predict the presence of heart disease (angiographic disease status). Two scans are run against the models that frame the use case as a partitioned and ordered multiclass classfication tasks respectively. | Multiclass classification | SVM <br /> Logistic Regression  | Robustness <br /> Explainability <br /> Explanation <br /> Fairness <br /> Performance | Ordered Multiclass <br /> Partitioned Multiclass |
 | [UnorderedMulticlass.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/multiclass) | Illustrates the use of Certifai to create and run a scan for an unordered multiclass-classification use case where the outcome classes are neither favorable nor unfavorable. | Multiclass classification | SVM <br /> Logistic Regression  | Robustness <br /> Explainability <br /> Performance <br /> | Unordered Multiclass |
+| [patient_readmission_xxx.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/patient_readmission) | A healthcare example illustrating counterfactual explanations and trust score scanning, using data that is one-hot encoded. Explanations and scores are viewed in the Console and analyzed in a notebook. | Binary Classification | MLP Classifier <br /> Logistic Regression | Explanation <br /> Fairness | One-hot encoding <br /> Save/load scan definition <by /> Preflight checks  |
```

Comment:
There's a typo in the last line-break tag right before "Preflight checks" in the last column. Not sure how the typo is handled (i guess github just ignores it?) because the text looks fine in the rendered markdown